http/main.go

```diff
@@ -62,26 +73,40 @@ func (h *httpNotifier) SendNotification(ctx context.Context, build *cbpb.Build)
 
 	log.Infof("sending HTTP request for event (build id = %s, status = %s)", build.Id, build.Status)
 
+	bindings, err := h.br.Resolve(ctx, nil, build)
+	if err != nil {
+		return fmt.Errorf("failed to resolve bindings: %w", err)
+	}
+	h.tmplView = &notifiers.TemplateView{
+		Build:  &notifiers.BuildView{Build: build},
+		Params: bindings,
+	}
+
 	logURL, err := notifiers.AddUTMParams(build.LogUrl, notifiers.HTTPMedium)
 	if err != nil {
 		return fmt.Errorf("failed to add UTM params: %w", err)
 	}
 	build.LogUrl = logURL
 
-	mo := protojson.MarshalOptions{}
-	jb, err := mo.Marshal(proto.MessageV2(build))
+	payload := new(bytes.Buffer)
+	var buf bytes.Buffer
+	if err := h.tmpl.Execute(&buf, h.tmplView); err != nil {
+		return err
+	}
+	err = json.NewEncoder(payload).Encode(buf)
 	if err != nil {
-		return fmt.Errorf("failed to marshal Build proto to JSON: %w", err)
+		return fmt.Errorf("failed to encode payload: %w", err)
 	}
-
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, h.url, bytes.NewReader(jb))
+	log.Infof("request body: %v", buf.String())
```

Comment:
why do we want to print this out? remove?

Reply:
Removed